### PR TITLE
a new feature

### PR DESCRIPTION
**Pull request by ostald**

### CODE REVIEW

File: olisfeature.txt [hunk 1]
```diff
@@ -0,0 +1 @@
+lala
```
Comment (Devin-Huyghebaert) [inline olisfeature.txt:1]: A very interesting feature

**Review by Devin-Huyghebaert — APPROVED** [review]

**Review by Sbr077 — APPROVED** [review]
oh

**Review by ingeborgfroystein — APPROVED** [review]

File: olisfeature.txt (hunk 1 above)
Comment (ostald) [reply olisfeature.txt:1]: hope it proves useful to others, too

Comment (AndreasKvammen) [reply olisfeature.txt:1]: good work!

**Pull request merged**